org.bunkus.mkvtoolnix-gui.yaml

```diff
@@ -24,12 +24,12 @@ modules:
   - name: boost
     buildsystem: simple
     build-commands:
-      - ./bootstrap.sh --prefix=/app --with-libraries=regex,filesystem,system,date_time
+      - ./bootstrap.sh --prefix=/app --with-libraries=regex,filesystem,system
```

Comment:
While you're at it: Boost's `regex` library hasn't been used for quite a while. Care to remove it, too?

And any particular reason you're adding libdvdread but not libdvdcss?

Reply:
> And any particular reason you're adding libdvdread but not libdvdcss?

It was not mentioned in https://www.bunkus.org/blog/2020/05/mkvtoolnix-v47-0-0-relesaed explicitly.

Since https://github.com/flathub/org.videolan.VLC and https://github.com/flathub/org.gnome.Totem are using it, I guess it's ok from a legal point.